<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据备份与恢复</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>                         数据备份软件</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>源路径:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_Source"/>
   </item>
   <item>
    <widget class="QPushButton" name="button_BrowseSource">
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>备份路径:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_Backup"/>
   </item>
   <item>
    <widget class="QPushButton" name="button_BrowseBackup">
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_Backup">
     <property name="text">
      <string>备份</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_Restore">
     <property name="text">
      <string>恢复</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
